api-gateway/src/routes/authRoutes.ts

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware';
+import config from '../config';
+import logger from '../utils/logger';
+import { authMiddleware } from '../middleware/authMiddleware';
+
+const router = express.Router();
+
+const authServiceProxy = createProxyMiddleware({
+  target: config.services.auth,
+  changeOrigin: true,
+  pathRewrite: {
+    '^/(.*)': '/api/v1/auth/$1', // Maps /login -> /api/v1/auth/login
+  },
+  on: {
+    proxyReq: fixRequestBody, // need to fix the request body before forwarding
+  },
+});
+
+// Log incoming requests to the Auth route
```

Comment:
Are we meant to use the logger here? If the logger is important then maybe we want to enable it

Reply:
i think its ok, if we enable logging it'll flood the terminal bc itll be a ton of incoming messages